src/Error/Debugger.php

```diff
@@ -442,6 +442,9 @@ public static function formatTrace($backtrace, array $options = [])
             if ($options['format'] === 'points' && $trace['file'] !== '[internal]') {
                 $back[] = ['file' => $trace['file'], 'line' => $trace['line']];
             } elseif ($options['format'] === 'array') {
+                if (!$options['args']) {
+                    unset($trace['args']);
```

Comment:
Should we fix wherever args is checked?

Reply:
We could move this up to around here: https://github.com/cakephp/cakephp/blob/15448de161da53f22ecab0409f2b4d3a7020d749/src/Error/Debugger.php#L419

As far as I cold tell its only a problem when the format is array.